docs/Makefile

```diff
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
@fridex is there a preferred task runner in the python community? Right now we're using `Makefile` for most of our stuff, because I'm old

Reply:
Ah, seems that this is created by shpinx? Nevertheless, the question about the task runner still stands :)

Reply:
In thoth we use something like:

python3 setup.py build_sphinx

Sphinx needs to be installed, the command above looks for doc or docs directory with sources

Reply:
https://www.sphinx-doc.org/en/master/usage/advanced/setuptools.html

https://github.com/thoth-station/nepthys/blob/master/create_docs.sh

https://github.com/thoth-station/adviser/blob/c68cd3841684380cfb3ba2468b5c3976938d6e84/setup.py#L64